src/HasStatuses.php

```diff
@@ -71,6 +71,29 @@ public function scopeCurrentStatus(Builder $builder, string $name)
             });
     }
 
+    /**
+     * @param string|array $names
+     *
+     * @return void
+     **/
+    public function scopeNotCurrentStatus(Builder $builder, ...$names)
```

Comment:
Nice! Could you rename this to `otherCurrentStatus`?

Reply:
Hey @freekmurze! Yah I was actually thinking about this after I submitted the PR..

I can definitely rename it to `otherCurrentStatus` but would something like `withoutCurrentStatus` make more sense?